stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/StreamVideoBuilder.kt

```diff
@@ -84,6 +86,7 @@ public class StreamVideoBuilder @JvmOverloads constructor(
     private val videoDomain: String = "video.stream-io-api.com",
     private val runForegroundServiceForCalls: Boolean = true,
     private val localSfuAddress: String? = null,
+    private val sounds: Sounds = Sounds(),
```

Comment:
I've added this `sounds` parameter in `StreamVideoBuilder`. I then pass it to `StreamVideoImpl` so it can be used wherever needed. It was the closest I could get to the iOS implementation.

iOS uses an [Appearance](https://github.com/GetStream/stream-video-swift/blob/main/Sources/StreamVideoSwiftUI/Appearance.swift) class to customize the sounds. It passes it to a [StreamVideoUI](https://github.com/GetStream/stream-video-swift/blob/main/Sources/StreamVideoSwiftUI/StreamVideoUI.swift) class, which is used to initialize the video client, as described [here](https://getstream.io/video/docs/ios/basics/quickstart/#setting-up-the-streamvideoui-object).

The Sounds class is identical to the [iOS one](https://github.com/GetStream/stream-video-swift/blob/main/Sources/StreamVideoSwiftUI/Sounds.swift).

Is this way of providing sounds ok? Another way would have been to use the Compose Theme, but we're using the sounds in `CallService`.